coco/ccml/parse.py

```diff
@@ -9,6 +11,35 @@
 SPECIFIC_TAG_REGEX = "<(\/|)({0})(|[^\<\>]+)>"
 
 
+def escape_special_characters(text):
```

Comment:
good chance there is a built in fucntion in lxml / python xml to escape xml strings

Reply:
Updated, using it instead of my manual replacements, cause if I escape the whole xml text it escape the xml tags and doest work properly.